src/core/plugins/oas3/fn.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @prettier
+ */
+
+export const getIsFileFormat = (schema) => {
+  if (!schema) return false
+
+  const isStringType = schema.get("type") === "string"
+  const format = schema.get("format")
+  const isBinaryFormat = format === "binary"
+  const isBase64Format = format === "base64" || format === "byte"
```

Comment:
Comment author @glowcloud:

Until now, we always checked for `base64` format but not for `byte`. 
In 3.0.3, the [data type](https://github.com/swagger-api/swagger-ui-plus/blob/77fd7178f7b6b7c97d3d0e881db14a7d5e4f59ab/src/core/plugins/json-schema-2020-12/fn.js#L68) for base64 encoded characters is `byte` but when there are examples for [file uploads](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.3.md#considerations-for-file-uploads), it specifies `format: base64`.
In 3.0.4, the data type still specifies `byte` and [one of the examples](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.4.md#example-url-encoded-form-with-binary-values) specifies `format: byte`. There is no example with `format: base64`.
Not sure if we should just drop `base64`.